app/src/main/res/values/dimens.xml

```diff
@@ -31,7 +35,10 @@
     <dimen name="margin_large_n">-24dp</dimen>
     <dimen name="margin_xlarge_n">-32dp</dimen>
 
+    <dimen name="value_padding">10dp</dimen>
```

Comment:
value for what padding ?

Reply:
Updated in 4b9217e98b91d1f461395d2ec7cf4637c9e2c293